src/containers/forms/PractitionerForm/tests/index.test.tsx

```diff
@@ -0,0 +1,34 @@
+import { mount, shallow } from 'enzyme';
```

Comment:
Do you feel that these tests offer us full coverage of this component and its usage?

Reply:
not currently, i was hoping to do that as a different issue, since i think it might take awhile, created #526